vendor/modules.txt

```diff
@@ -130,6 +130,8 @@ github.com/mitchellh/go-homedir
 github.com/modern-go/concurrent
 # github.com/modern-go/reflect2 v1.0.1
 github.com/modern-go/reflect2
+# github.com/openshift/api v3.9.0+incompatible
```

Comment:
Is this intentional? If so, can we just remove it?

Reply:
This is needed because it contains the `Route` type